docs/examples/driver_examples/Qcodes example with Keysight B1500 Parameter Analyzer.ipynb

```diff
@@ -807,6 +807,194 @@
     "constants.ComplianceStatus.C"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Performing open/phase compensation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Before performing these compensation/calibration measurements make sure that **probe needle is not connected to the device**. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Phase compensation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Before performing phase compensation make sure that all the channel outputs are enabled else instrument throws an error. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 29,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "b1500.cmu1.enable_outputs()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Also the phase compensation mode must set to manual. These two are the requirements for running the phase compensation.  "
```

Comment:
hm, i'd suggest to put this phrase "These two are the requirements for running the phase compensation." to the beginning of the section for clarity.